<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences ..</string>
  </property>
  <property name="windowIcon">
   <iconset resource="KanmusuMemory.qrc">
    <normaloff>:/resources/KanmusuMemory32.png</normaloff>:/resources/KanmusuMemory32.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Pictures path</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="savePathEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="selectPathButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="savePngCheckBox">
       <property name="text">
        <string>Save in PNG format.</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="unusedTwittercheckBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Unused Twitter</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="maskAdmiralNameCheckBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Mask Admiral name</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="maskHqLevelCheckBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Mask HQ level</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="timerAutoStartCheckBox">
       <property name="text">
        <string>Auto Start (Provisional)</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Buttle result</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="viewButtleResultCheckBox">
       <property name="text">
        <string>View buttle result</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Opacity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="buttleResultOpacityComboBox">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>100%</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>75%</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>50%</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>25%</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="10" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Position</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="buttleResultPositionComboBox">
         <property name="currentText">
          <string>RightTop</string>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <item>
          <property name="text">
           <string>LeftTop</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RightTop</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>LeftBottom</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RightBottom</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Center</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="11" column="1">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="QCheckBox" name="disableExitShortcut">
       <property name="text">
        <string>Disable exit(Ctrl+Q) shortcut</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="proxyHostLabel_3">
       <property name="text">
        <string>Browser:</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QCheckBox" name="useCookieCheckBox">
       <property name="text">
        <string>Use Cookie(need to restart the Kanmemo)</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QCheckBox" name="disableContextMenu">
       <property name="text">
        <string>Disable coontext menu</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="proxyHostLabel_2">
       <property name="text">
        <string>Proxy:</string>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="QCheckBox" name="proxyEnableCheckBox">
       <property name="text">
        <string>Enable</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="proxyHostLabel">
         <property name="text">
          <string>Proxy host:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="proxyHost"/>
       </item>
      </layout>
     </item>
     <item row="19" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="proxyPortLabel">
         <property name="text">
          <string>Proxy port:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="proxyPort">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="proxyPortMergin">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Timer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>savePathEdit</tabstop>
  <tabstop>selectPathButton</tabstop>
  <tabstop>savePngCheckBox</tabstop>
  <tabstop>unusedTwittercheckBox</tabstop>
  <tabstop>maskAdmiralNameCheckBox</tabstop>
  <tabstop>proxyHost</tabstop>
  <tabstop>proxyPort</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="KanmusuMemory.qrc"/>
 </resources>
 <connections/>
</ui>
